File: Gene/Engine/Private/Content/GeneModel.cc
#include <Content/GeneModel.h>
#include <Math/Vector3.h>
using namespace Gene::Content;

GeneModel::GeneModel(std::vector<Math::Vector3> vertices,
					 std::vector<Math::Vector3> normals, 
				     std::vector<GLuint> indices) 
{
	Vertices = vertices;
	Normals = normals;
	Indices = indices;
}

File: Tools/ModelConverter/Source/GeneModel.cs
using System;
using System.Collections.Generic;
using System.Text;
using ModelConverter.Source.Utils;

namespace ModelConverter.Source
{
    class GeneModel
    {
        public List<Vector3> Vertices = new List<Vector3>();
        public List<Vector3> Normals = new List<Vector3>();
        public List<Vector2> UVs = new List<Vector2>();
        public List<int> Indices = new List<int>();
    }
}
